standard/tokens/jettons/how-to-find.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: "How to find Jetton wallet"
+---
+
+To compute the address of a Jetton wallet from the address of its owner (a regular user wallet),
```

Comment:
The intro should explain _why_ we need to compute a Jetton wallet address. What higher-level problem are we solving?